db/migrate/20201014082050_add_welsh_flag_to_ldu.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class AddWelshFlagToLdu < ActiveRecord::Migration[6.0]
+  # Need to define this for Heroku review apps which run all the migrations
+  class LocalDivisionalUnit < ApplicationRecord
+  end
+
```

Comment:
Why does Heroku run all migrations rather than building the database from `schema.rb`?

This seems a bit weird? 🤔 Surely that means any time we remove/rename a model, we'll need to add this 'workaround' to keep Heroku happy. That seems wrong to me.

Reply:
It seems that it wants to 'update' the schema (seems reasonable) but for review apps the schema is empty - hence it runs all the migrations